README.md

```diff
@@ -55,7 +55,7 @@ Simple example for Kotlin/JVM
 ```kotlin
 fun main() {
     val skiaLayer = SkiaLayer()
-    skiaLayer.skikoView = GenericSkikoView(skiaLayer, object : SkikoView {
+    skiaLayer.renderDelegate = SkiaLayerRenderDelegate(skiaLayer, object : SkikoRenderDelegate {
```

Comment:
Maybe `SkikoRenderDelegate` -> `SkikoRenderer`?

Reply:
This entity is a helper function used only in demo and ideally at sometime (when we'll have a single demo project in skiko) will move there.